### ISSUE TITLE
Figure out a plan for storing, aggregating, and querying server logs

### ISSUE THREAD

**Issue by chuckharmston** [mozilla/universal-search-recommendation#105]
Likely plan: S3 -> Modified (not now) -> Redshift

**Comment by wresuolc** [comment]
Alright, so, data should be going into S3 now, but we need to get it into re:dash to be [easily] query-able.  To do that, we need to tell re:dash what columns are what, otherwise you can't run queries on it.

Rebecca is going to write a script which will convert the data we're logging into a CSV which re:dash will import.  That is being tracked at https://bugzilla.mozilla.org/show_bug.cgi?id=1267838

Once that is done, we'll be good to go.

**Comment by chuckharmston** [comment]
@clouserw / @wresuolc: Do we need to change the logger name, or are we all good here?

**Comment by wresuolc** [comment]
We might need to.  I'm not sure yet

**Comment by wresuolc** [comment]
Bugs should be closed by EOD

**Comment by 6a68** [comment]
@wresuolc Any updates? Can we close this pup?

**Comment by wresuolc** [comment]
Next steps @ https://bugzilla.mozilla.org/show_bug.cgi?id=1270586

**Comment by wresuolc** [comment]
I think I misled us on this one.  The bug mentioned above in my first comment is closed, which is what we were waiting on here.  The bug in my last comment is about telemetry which is still an issue, however, this issue specifically says "server logs" so I think it's done.  Server logs are queryable at https://sql.telemetry.mozilla.org/queries/new under the Testpilot data source.
